src/main/scala/com/gu/fulfilmentLookup/Lambda.scala

```diff
@@ -6,27 +6,37 @@ import java.time.format.DateTimeFormatter
 import play.api.libs.json.{ JsError, JsSuccess, JsValue, Json }
 import com.amazonaws.services.lambda.runtime.Context
 import com.gu.fulfilmentLookup.ResponseWriters._
-import scala.util.{ Failure, Success }
+import scala.util.{ Failure, Success, Try }
 import scalaz.{ -\/, \/- }
 
 trait FulfilmentLookupLambda extends Logging {
 
-  def s3Client: CsvClient
+  def csvClient: CsvClient
   def caseService: CaseService
-  def config: Config
+  def stage: String
+  def loadConfig: Try[Config]
 
   // Entry point for the Lambda
   def handler(inputStream: InputStream, outputStream: OutputStream, context: Context): Unit = {
-    logger.info(s"Fulfilment Lookup Lambda is starting up...")
+    logger.info(s"Fulfilment Lookup Lambda is starting up in $stage")
     val inputEvent = Json.parse(inputStream)
     val maybeBody = (inputEvent \ "body").toOption
     maybeBody match {
       case Some(body) => {
         Json.fromJson[LookupRequest](Json.parse(body.as[String])) match {
           case validLookup: JsSuccess[LookupRequest] => {
             logger.info(s"Received request the following data: ${validLookup.value}")
-            val response = lookUp(validLookup.value, outputStream)
-            outputForAPIGateway(outputStream, Json.toJson(response))
+            loadConfig match {
+              case Success(config) => {
+                val response = lookUp(config, validLookup.value, outputStream)
+                outputForAPIGateway(outputStream, Json.toJson(response))
+              }
+              case Failure(error) => {
+                logger.error(s"Failed to load config in $stage due to $error")
```

Comment:
will we get a sentry error if this happens?

Reply:
Not at the moment... all of these lambda-based projects (including fulfilment-lambdas, I think?) need to be configured with Sentry/Kibana and have a more general monitoring strategy implemented